SportShop/Model/Product.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SportShop.Model
+{
+    public class Product : INotifyPropertyChanged
+    {
+        public static int counter = 1;
+
+        private string productName;
+
+        private string productModel;
+
+        private string countryDistributor;
+
+        private decimal price;
+
+        private string sportCategory;
+
+        private string image;
+
+        public int Id { get; set; }
+
+        public string ProductName
+        {
+            get { return productName; }
+            set
+            {
+                productName = value;
+                OnPropertyChanged("ProductName");
```

Comment:
We should not use literals. Please use nameof(this.ProductName) instead in all cases.

Reply:
-1 to source code quality.